spec/models/concerns/arclight/solr_document_spec.rb

```diff
@@ -86,11 +86,27 @@
     end
   end
 
-  describe '#normalize_date' do
```

Comment:
@gordonleacock it looks like the coverage decrease is because you're removing this test for the `normalize_date`